snap/hooks/install

```diff
@@ -5,8 +5,8 @@ set -eux
 # Create the necessary parent directories 
 mkdir -p "${SNAP_DATA}/etc/pbm/"
 mkdir -p "${SNAP_DATA}/etc/mongod/"
-mkdir -p "${SNAP_COMMON}/var/lib/mongod/"
-mkdir -p "${SNAP_COMMON}/var/log/mongod/"
+mkdir -p "${SNAP_COMMON}/var/lib/mongodb/"
+mkdir -p "${SNAP_COMMON}/var/log/mongodb/"
```

Comment:
```suggestion
# data related to mongodb workload stored here
mkdir -p "${SNAP_COMMON}/var/lib/mongodb/"
# metadata related to mongod should be stored separately 
mkdir -p "${SNAP_COMMON}/var/lib/mongod/"
mkdir -p "${SNAP_COMMON}/var/log/mongod/"
```
We want both `"${SNAP_COMMON}/var/lib/mongodb/"``"${SNAP_COMMON}/var/lib/mongod/"`

[motivation here](https://github.com/canonical/mongodb-operator/pull/176#discussion_r1129784914)